ee/ucp/kubernetes/storage/use-azure-files.md

```diff
@@ -287,7 +285,5 @@ subjects:
 
 ## Where to go next
 
-- [Deploy an Ingress Controller on
-  Kubernetes](/ee/ucp/kubernetes/layer-7-routing/)
-- [Discover Network Encryption on
-  Kubernetes](/ee/ucp/kubernetes/kubernetes-network-encryption/)
+- [Deploy an Ingress Controller on Kubernetes](/ee/ucp/kubernetes/layer-7-routing/)
+- [Discover Network Encryption on Kubernetes](../kubernetes-network-encryption.md)
```

Comment:
Wondering why we have a relative path and an absolute path for topics in the same directory

Reply:
Looked at some of those (probably worth a discussion after ee-topics have migrated). So:

Advantage of relative links:

- they work when browsing the markdown on GitHub (I think absolute links don't work there, but would have to double-check)
- moving a whole directory to a different location doesn't break links to pages in the same directory
- ^^ this could help with, e.g. moving whole of `ee/ucp` topic to an archived location (`datacenter/ucp/3.x`), and still linking to related topics for the _same_ version (not the "current" version)

Disadvantage of relative links:

- they can be harder to read. it's "ok" when linking to a child directory (`child-dir/page.md`), but becomes hard to follow when linking to parent directories (`../../../somedir/page.md`)
- it's hard to find/replace links when things move (can't search for `/some/old/dir/page.md` to find which links needs updating)


I think the disadvantages are real; mostly when linking to "other" topics (linking to, e.g., a reference-docs page from a product manual). I'm ok with using relative links to link within the same topic though.

Reply:
Sure, thanks for that. Agree, relative links can be very confusing when linking to parent directories. We can discuss this in one of our weekly syncs.